client/packages/cli_new/src/schemas/transfer.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from "zod"
+
+export const TransferSideEffectSchema = z.object({
```

Comment:
This is not Transfer specific, so we can rename to SideEffectSchema

Reply:
Fixed; please refer to commit cdc64d300e9ae36c3e2973b2f5110c73381f3530.